.gitignore

```diff
@@ -10,6 +10,7 @@ project.xcworkspace
 
 # build files generated by the configure script
 *.ninja
+*.ninja~
```

Comment:
Since we're making other changes to this script, I'd suggest putting *~ at the top of the file and removing this line. That way any backup file won't get committed. It's a fairly standard thing to have in Linux repos, but tends to be forgotten on Mac ones.

See e.g. https://github.com/github/gitignore/blob/master/Global/Linux.gitignore#L1